### PR TITLE
Initial commit for TimeSeries

### PR DESCRIPTION
**Pull request by rajivshringi**
Model for netflix timeseries. Test verification pending.

### CODE REVIEW

File: service_capacity_modeling/models/org/netflix/time_series.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+    @staticmethod
+    def extra_model_arguments() -> Sequence[Tuple[str, str, str]]:
+        return nflx_java_app_capacity_model.extra_model_arguments()
+
+    @staticmethod
+    def compose_with(
+        user_desires: CapacityDesires, extra_model_arguments: Dict[str, Any]
+    ) -> Tuple[str, ...]:
+        # In the future depending on the user desire we might need EVCache
```
Comment (jolynch) [inline service_capacity_modeling/models/org/netflix/time_series.py:59]: nit: depending on the extra model arguments we'll need to add Elasticsearch

File: tests/netflix/test_time_series.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+from service_capacity_modeling.capacity_planner import planner
+from service_capacity_modeling.interface import CapacityDesires
+from service_capacity_modeling.interface import DataShape
+from service_capacity_modeling.interface import Interval
+from service_capacity_modeling.interface import QueryPattern
+
+
+def test_timeseries_increasing_qps_simple():
+    qps_values = (100, 1000, 10_000, 100_000)
```
Comment (jolynch) [inline tests/netflix/test_time_series.py:9]: Since this is a pretty slow test to run on every model we could just have 1k, 10k, 100k

**Review by jolynch — APPROVED** [review]

File: service_capacity_modeling/models/org/netflix/time_series.py (hunk 1 above)
Comment (rajivshringi) [reply service_capacity_modeling/models/org/netflix/time_series.py:59]: How do you envision the arguments to be? Would that be an enum that ties to a model name or the model name itself? Should we add that in the next iteration when we are adding support for ES?

File: tests/netflix/test_time_series.py (hunk 2 above)
Comment (rajivshringi) [reply tests/netflix/test_time_series.py]: Updated

File: service_capacity_modeling/models/org/netflix/time_series.py (hunk 1 above)
Comment (jolynch) [reply service_capacity_modeling/models/org/netflix/time_series.py:59]: The extra model arguments come from the free-form parameters. So I imagine that in the UI we'd have "Require best-effort cross event search?" and that would set a boolean flag "search=True" or some such in extra_model_arguments.

Then this function can return both elastic and C* when we are composing them.

**Review by jolynch — APPROVED** [review]

**Pull request merged**